### PR TITLE
Fixed typo in README.md database creation instructions.

### PR DESCRIPTION
**Pull request by hiramv**
Noticed a small typo in the installation instruction in README.md, under the 'Database creation' part. The command shown is now 'rake db:migrate' instead of 'rake db:migration'.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ To install every dependency:
 
 Database creation (Development Environment):
 
-    $ bin/rake db:migration
+    $ bin/rake db:migrate
```
Comment (hiramv) [inline README.md:31]: fixed typo

**Comment by jonahoffline** [comment]
Hello, thanks for contributing Hiram! Can you remove the imagemagick `*.dylib` files and `Gemfile.lock` so I can merge the pull request? It appears that a few other files got added to the repo. Thanks :+1:

**Comment by hiramv** [comment]
no problem. acabo de quitar los files.

**Comment by jonahoffline** [comment]
Awesome, gracias! :alien: :+1:

**Pull request merged**